vyked/jsonprotocol.py

```diff
@@ -85,6 +102,9 @@ def data_received(self, byte_data):
             # recover from invalid data
             self.logger.exception('Invalid data received')
             self.set_streamer()
+        
+        if json_parse and element['type'] in ['request', 'response']:
+            self.logger.debug("{} Packet Endpoint: {}  Json Loads Time: {} ms".format(element['type'], element['endpoint'], (time.time() - start_time) * 1000))
```

Comment:
Lets also log, how many times json.loads was called for each request.
I suppose bigger data would increases the number of times loads is called, and each time it will be time consuming. U can also log cumulative time taken by all json loads